test/verify/check-superuser

```diff
@@ -77,6 +77,7 @@ class TestSuperuser(MachineCase):
         b.leave_page()
         b.check_superuser_indicator("Administrative access")
 
+    @nondestructive
```

Comment:
OOI, at first sight I don't see anything in testBasic which would be destructive? Possibly you could make the whole class `@nond`? (Follow-up material)

Reply:
What about,
```
          m.execute("rm -rf /var/{db,lib}/sudo/lectured/admin")
```

Reply:
It's not super important to restore it; but in principle, you could use `self.restore_dir()` or `restore_file()` and replace the braces with a conditional or two calls.

Reply:
Okido fixed.